src/components/navigation.tsx

```diff
@@ -15,10 +22,18 @@ interface Props {
     faq: string;
     contact: string;
   };
+  languageLabels: {
```

Comment:
Is there a way to generalize the language labels (rather than hard-coding these in the interface)? Then there is one 'source of truth' for the locales (e.g. the files in `messages`) and we don't have to update this interface when we need to add another locale?

Reply:
The source of truth is now the configuration in the middleware.